pom.xml

```diff
@@ -139,38 +139,38 @@
     <dependency>
       <groupId>com.googlecode.netlib-java</groupId>
       <artifactId>netlib-java</artifactId>
-      <version>1.0</version>
-    </dependency>
-    <dependency>
-      <groupId>commons-io</groupId>
-      <artifactId>commons-io</artifactId>
-      <version>2.7</version>
+      <version>1.1</version>
```

Comment:
Is this version downgrade expected?

Reply:
It is not a version downgrade. I removed commons-io from here and added it later in the file. I wanted it to be upgraded after spark-mllib because it is a dependency and could be overwritten by an older version.

Reply:
That sounds correct. thanks